src/views/Blog.vue

```diff
@@ -74,6 +83,33 @@ export default {
   },
   methods: {
     get,
+    jsPDF,
+
+    async getPDF() {
+      this.$refs.pdfContent.innerHTML = this.$refs.blog.innerHTML;
+      const page = this.$refs.pdfContent;
+      const filename = get(this.blog, 'attributes.title', 'blog');
+
+      var doc = new jsPDF({
+        orientation: "p",
+        format: "a4",
+        unit: "px"
+      });
+
+      await doc.html(page, {
+        html2canvas: {
+          scale: 0.38
+        },
+        callback: function (doc) {
+          doc.save(`${filename}.pdf`);
+        },
+        x: 76.8,
```

Comment:
Would suggest saving these into variables.
It's a bit unclear what these numbers mean. 
For example if we were to review this code in a couple of months, we would scratch our heads trying to understand what these numbers mean